### ISSUE TITLE
Failed to load translation file error

### ISSUE THREAD

**Issue by Kmarconi** [retifrav/translating-qml#5]
Hi !

First, thanks for posting this code example for dynamic translation. I just one question : Do you know why not matter I try, I always have the error from the Trans.cpp file : Failed to load translation file, falling back to English. Is there something wrong in the path used ? I'm using Qt 5.12.3 with the retranslate update. 

Thanks !

**Comment by retifrav** [comment]
Yes, it could be because of wrong path or missing translation files. I've seen that error more often on Mac OS, as there it's a bit more complicated with application bundles.

**Comment by Kmarconi** [comment]
I'm on Ubuntu and I haven't touched your code, I've just downloaded it and open the pro file in my qt creator. Is there anything else I should do ? main.qml and styles.qml are greyed out in my file tree, is it normal ?

**Comment by retifrav** [comment]
As it says in the [article](https://retifrav.github.io/blog/2017/01/04/translating-qml-app/), you need to put the translation files to where the application can find them.

**Comment by Kmarconi** [comment]
Yes I know but the app is looking in the i18n directory so it should find it
